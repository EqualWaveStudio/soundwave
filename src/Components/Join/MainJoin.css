p {
    color: #E01885;
  }
  span {
    color: white;
  }
  
  body{
      background-color: var(--dark-blue);
      margin: 0;
      height: 100vh;
      font-size: 1.5rem;
      color: white;
  }
  main{
    width: 100vw;;
    height: 100%;
    overflow: hidden;  
  h1{
      font-weight: 400;
      font-size: 3.9rem;
      }
  
  }
  .content{
      width: 100%;
      height: 100%;
      position: absolute;
      display: flex;
      justify-content: space-evenly;
      align-items: flex-end;
      overflow: hidden;
  }
  
  .text {
      width: 50vw;
      margin-bottom: 15rem;
      padding: 0 5rem;
      color: white; 
      z-index: 1;
      
    }
    
  .pink-text {
      color: var(--pink-dark);
  
  }
  
  .container div{
      justify-content: left;
      align-items: left;    
      flex-direction: column;
      border-radius: 10px;
      padding: 1vw;
      margin-bottom: 1vw;
      margin-top: 1vw;
      margin-left: 2.5vw;
      margin-right: 2.1vw;
      
  
    }
  
  .container {
      width: 30%;
      padding-bottom: 3%;
      background-color: var(--dark-purple);
      position: absolute;
      top: 25%;
      left: 55%;
      border-radius: 5%;
      font-size: 1vw;
      align-items: center;
    
  }
  
  .form-control .label-text {
    text-align: left;
  }
  
  .button {
      background-color: var(--blue-btn);
      color: white;
      padding: 1.3vh;
      margin-top: .5rem;
      border-radius: 0.5rem;
      border: none;
      font-weight: bold;
      cursor: pointer; 
      font-size: 2vh;
      width: 76%;   
      display: block;
      margin: auto;
  
  
  }
    
  .button:hover {
    background-color: var(--pink-dark);
    color: var(--dark-purple);
  }
  
  .pink-circle{
      background-color: var(--dark-pink);
      border-radius: 50%;
      position: absolute;
      top: 20%;
      left: -13%;
      opacity: 1;
      width: 45%;
      padding-bottom: 45%;
  
  }
  
  .purple-circle{
      background-color: var(--dark-purple);
      border-radius: 50%;
      position:absolute;
      bottom: 55%;
      left: 25%;
      opacity: 0.3;
      width: 30%;
      padding-bottom: 30%;
      
  }
  
  
  
  .transparent-input {
      background-color: transparent;
      border: 1px solid var(--purple);
      border-radius: 0.3vw;
      padding: 1vh 6.3vh 1vh;
      width: 80%;
      outline: none;
      color: white;
      padding-left: 1vw;
          
    }
  
    small {
      color: var(--pink-dark);
    }
      