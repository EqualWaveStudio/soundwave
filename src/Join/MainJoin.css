
.formText {
  color: #E01885;
}
span {
  color: white;
}

* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

h1 {
    font-weight: 400;
    font-size: 3.9rem;
    }

@media only screen and (max-width: 390px) {
  h1 {
    
   }
 }

.content{
    width: 100%;
    height: 100%;
    position: absolute;
    display: flex;
    justify-content: space-evenly;
    align-items: flex-end;
    overflow: hidden;
}
@media only screen and (max-width: 390px) {
  .content  {
    width: 50vh;
    display: flex;
    align-items: center;
    flex-direction: column;

  }
}

.text {
    width: 50vw;
    margin-bottom: 35rem;
    padding: 0 5rem;
    color: white; 
    z-index: 1;
  }
  @media only screen and (max-width: 390px) {
    .text {
      position: absolute;
      left: 10vh;
      padding: 6px;
      
    }
  }

.pink-text {
    color: var(--pink-dark);

}

.container div{
    justify-content: left;
    align-items: left;    
    flex-direction: column;
    border-radius: 10px;
    padding: 1vw;
    margin-bottom: 1vw;
    margin-top: 1vw;
    margin-left: 2.5vw;
    margin-right: 2.1vw;
  }

.container {
    width: 30%;
    padding-bottom: 3%;
    background-color: var(--dark-purple);
    position: absolute;
    top: 25%;
    left: 55%;
    border-radius: 5%;
    font-size: 1vw;
    align-items: center;
  
}
@media only screen and (max-width: 390px) {
  .container {
    margin-top: 20vh;
    position: absolute;
    margin-left: -22vh;
    width: 40vh;
    font-size: 4vw;
    align-items: center;
  }
}

@media only screen and (max-width: 390px) {
  .content {
    width: 100%;
    height: 100%;
  }
}
.form-control .label-text {
  text-align: left;
}

.button {
    background-color: var(--blue-btn);
    color: white;
    padding: 1.3vh;
    margin-top: .5rem;
    border-radius: 0.5rem;
    border: none;
    font-weight: bold;
    cursor: pointer; 
    font-size: 2vh;
    width: 76%;   
    display: block;
    margin: auto;
}
  
.button:hover {
  background-color: var(--pink-dark);
  color: var(--dark-purple);
}

.pink-circle{
    background-color: var(--dark-pink);
    border-radius: 50%;
    position: absolute;
    top: 20%;
    left: -13%;
    opacity: 1;
    width: 45%;
    padding-bottom: 45%;

}

.purple-circle{
    background-color: var(--dark-purple);
    border-radius: 50%;
    position:absolute;
    bottom: 55%;
    left: 25%;
    opacity: 0.3;
    width: 30%;
    padding-bottom: 30%;
    
}



.transparent-input {
    background-color: transparent;
    border: 1px solid var(--purple);
    border-radius: 0.3vw;
    padding: 1vh 6.3vh 1vh;
    width: 90%;
    outline: none;
    color: white;
    padding-left: 1vw;
  }

  small {
    color: var(--pink-dark);
  }
    
