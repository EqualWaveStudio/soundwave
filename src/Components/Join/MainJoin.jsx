import React from 'react';
import Form from './Form';
import './MainJoin.css'


function MainJoin() {
  return (
    <main>

      <div className="content">

        <div className='text'>
          <h1>
            Join the <span className="pink-text">fun.</span>
          </h1>
        </div>

        <div className='pink-circle'></div>
        <div className='purple-circle'></div>

      </div>
      <div>
        <Form />
      </div>


    </main>

  );
}

  export default MainJoin;