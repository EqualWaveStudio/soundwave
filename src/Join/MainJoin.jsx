import React from 'react';
import '../Join/MainJoin';
import Form from '../Components/FormValidation';
import './MainJoin.css'


const MainJoin = () => {
    return(
          <main>
      
              <div className="content">
                        
              <div className='text'>
                <h2>
                Join the <span className="pink-text">fun.</span>
                </h2>
                 </div>
                
                <div className='pink-circle'></div>
                <div className='purple-circle'></div>
                                
              </div>
              <div> 
             <Form />
              </div>
              
            
          </main>
      
    )
      };

  export default MainJoin;
  
