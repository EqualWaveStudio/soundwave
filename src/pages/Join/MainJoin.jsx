import React from 'react';
import './MainJoin';
import Form from '../../Components/FormValidation/FormValidation';
import './MainJoin.css'
import '../../Components/FormValidation/FormValidation.css';
import Footer from '../../Components/Footer/Footer';


const MainJoin = () => {
    return(
      <>
          <main className='main-join'>
      
              <div className="content">
                        
              <div className='text'>
                <h2>
                Join the<span className="pink-text"> fun.</span>
                </h2>
                 </div>               
              </div>
              
             
              <Form />
          </main>
          <div className='pink-circle'></div>
          <div className='purple-circle'></div>

<Footer />
</>
      
    )
      };

  export default MainJoin;
  
