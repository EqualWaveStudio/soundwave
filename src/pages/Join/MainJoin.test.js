import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import MainJoin from './MainJoin';

describe('MainJoin', () => {
  test('renders correctly', () => {
    const tree = renderer.create(
      <BrowserRouter basename="/">
        <MainJoin />
      </BrowserRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
