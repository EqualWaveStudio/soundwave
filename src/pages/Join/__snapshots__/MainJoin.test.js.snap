// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainJoin renders correctly 1`] = `
Array [
  <main
    className="main-join"
  >
    <div
      className="content"
    >
      <div
        className="text"
      >
        <h2>
          Join the
          <span
            className="pink-text"
          >
             fun.
          </span>
        </h2>
      </div>
    </div>
    <div
      className="container"
    >
      <form
        onSubmit={[Function]}
      >
        <div
          className="form-control"
        >
          <label
            for="name"
          >
            Name:
          </label>
          <input
            className="transparent-input"
            id="name"
            name="name"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
          />
        </div>
        <div
          className="form-control"
        >
          <label
            for="email"
          >
            Email:
          </label>
          <input
            className="transparent-input"
            id="email"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
          />
        </div>
        <div
          className="form-control"
        >
          <label
            for="password"
          >
            Password:
          </label>
          <input
            className="transparent-input"
            id="password"
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            type="password"
          />
        </div>
        <button
          className="button"
          type="submit"
        >
          Join Now
        </button>
      </form>
    </div>
  </main>,
  <div
    className="pink-circle"
  />,
  <div
    className="purple-circle"
  />,
  <footer
    className="footer"
  >
    <div
      className="Footer-Left"
    >
      <a
        className="t-button hover-t-buttons"
        href="/discover"
        onClick={[Function]}
      >
        About Us
      </a>
      <a
        className="t-button hover-t-buttons"
        href="/join"
        onClick={[Function]}
      >
        Contact
      </a>
    </div>
    <div
      className="Footer-Right"
    >
      <div
        className="twitter"
      >
        <img
          alt="LogoTwitter"
          className="logoRs"
          src="vector(1).png"
        />
        <a
          className="rs-button"
          href="https://twitter.com/?lang=es/"
        >
          Twitter
        </a>
      </div>
      <div
        className="facebook"
      >
        <img
          alt="LogoFacebook"
          className="logoRs"
          src="vector(2).png"
        />
        <a
          className="rs-button"
          href="https://es-es.facebook.com/"
        >
          Facebook
        </a>
      </div>
    </div>
  </footer>,
]
`;
